Package.swift

```diff
@@ -13,7 +13,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-argument-parser.git", from: "1.4.0"),
-        .package(url: "https://github.com/maiqingqiang/Jinja", from: "1.0.0")
+        .package(url: "https://github.com/maiqingqiang/Jinja", "1.0.1"..<"1.1.0")
```

Comment:
Hmm, why is this necessary? If you run `Update to Latest Package Versions` in Xcode it retrieves `1.0.1`, as expected.

Reply:
Haha, I just found out that trick too! 😂
Thanks for the heads-up!